src/components/WebexLocalMedia/WebexLocalMedia.jsx

```diff
@@ -14,16 +14,20 @@ import {
 } from '../hooks';
 
 /**
- * Webex Local Media component displays the user's local video.
+ * Webex Local Media component displays the user's local video or local share.
+ * If the meeting is active and the current user is sharing the screen,
+ * the local share is displayed; otherwise, the local video is displayed (if video is enabled).
  *
  * @param {string} props.meetingID  ID of the meeting from which to obtain local media
  * @returns {Object} JSX of the component
  */
 export default function WebexLocalMedia({className, meetingID}) {
   const [mediaRef, {width}] = useElementDimensions();
-  const {localVideo} = useMeeting(meetingID);
+  const {localVideo, localShare, remoteVideo} = useMeeting(meetingID);
+  const isActive = remoteVideo !== null;
```

Comment:
@cipak, @alinasuciu don't forget to add a subtask to update this when refactoring meeting state

Reply:
This is not needed anymore as we moved the logic in InMeeting component.